src/i18n/strings/tchap_translations.json

```diff
@@ -265,5 +265,16 @@
   "<requestLink>Import from saved file</requestLink>": {
     "en": "<requestLink>Import from saved file</requestLink>",
     "fr": "<requestLink>Importer depuis le fichier sauvegardé</requestLink>"
+  },
+  "Export E2E room keys": {
+    "en": "Save your Tchap keys",
+    "fr": "Sauvegardez vos clés Tchap"
+  },
+  "Import E2E room keys": {
+    "en": "Import your Tchap keys from the downloaded file",
+    "fr": "Importez vos clés Tchap depuis le fichier sauvegardé"
+  },
+  "Back up your encryption keys with your account data in case you lose access to your sessions. Your keys will be secured with a unique Security Key.": {
```

Comment:
```suggestion
  "Back up your Tchap keys with your account data in case you lose access to your sessions. Your keys will be secured with a unique Security Key.": {
```

Reply:
I'd rather not modify the lable key